Sources/SwiftDocC/Infrastructure/DocumentationConverter.swift

```diff
@@ -166,7 +166,13 @@ public struct DocumentationConverter: DocumentationConverterProtocol {
         self.experimentalModifyCatalogWithGeneratedCuration = experimentalModifyCatalogWithGeneratedCuration
         
         // Inject current platform versions if provided
-        if let currentPlatforms {
+        if var currentPlatforms {
+            // Add missing platforms if their fallback platform is present.
+            DefaultAvailability.fallbackPlatforms.forEach { fallbackPlatform in
+                if (!currentPlatforms.keys.contains(fallbackPlatform.key.displayName)) {
+                    currentPlatforms[fallbackPlatform.key.displayName] = currentPlatforms[fallbackPlatform.value.rawValue]
+                }
+            }
```

Comment:
Is there a reason why `displayName` is used on the key but `rawValue` is used on the value (both are `PlatformName` elements)?

Also, this can be written simpler using a regular for loop that binds the key and values to separate meaningful names. If you want you can also move the if statement into a where clause on the loop
```suggestion
            for (platform, fallbackPlatform) in DefaultAvailability.fallbackPlatforms where currentPlatforms[platform.displayName] == nil {
                currentPlatforms[platform.displayName] = currentPlatforms[fallbackPlatform.rawValue]
            }
```

Reply:
For clarification, there are two pieces to my comment above:
- A question if there's a potential bug with using `.displayName` for one of the platforms and `.rawValue` for the other
- A non-blocking style suggestions about using a regular loop and using contextually relevant variable names instead of "key" and "value".

Reply:
I don't remember if the initial choice of using `rawValue` instead of `displayName` was intentional, but I don't think it makes a difference since in practice both will be the same (`iOS`).

However, after reviewing it, I decided to change it to `displayName` since it has the raw value as fallback and looks more consistent with the rest of the code.

```swift
self.displayName = displayName ?? rawValue
```